hera_cal/tests/test_abscal.py

```diff
@@ -234,6 +234,28 @@ def test_cut_bl(self):
         abscal.cut_bls(_data2, min_bl_cut=20.0, inplace=True)
         nt.assert_true(len(_data2), 12)
 
+    def test_dft_phase_slope_solver(self):
+        np.random.seed(21)
+
+        # build a perturbed grid
+        xs = np.zeros(100)
+        ys = np.zeros(100)
+        i = 0
+        for x in np.arange(0, 100, 10):
+            for y in np.arange(0, 100, 10):
+                xs[i] = x + 5 * (.5 - np.random.rand())
+                ys[i] = y + 5 * (.5 - np.random.rand())
+                i += 1
+
+        phase_slopes_x = (.2 * np.random.rand(5, 2) - .1)  # not too many phase wraps over the array
```

Comment:
what defines too many?

Reply:
If the phase wraps are so fast that we go through ~2 pi between grid points, the algorithm breaks down. It's basically an undersampling issue.

I clarified this in the comments.